src/App.js

```diff
@@ -1,33 +1,37 @@
 import React, { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
+import { SearchField } from './components/SearchField/SearchField';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  addQuery = (event) => {
+    this.setState({
+      query: event.target.value,
+    });
+  }
 
   render() {
+    const { query } = this.state;
+    const queryToLower = query.toLowerCase();
+
+    const choiceMovie = moviesFromServer.filter(
```

Comment:
```suggestion
    const chosenMovies = moviesFromServer.filter(
```
фильтр возвращает массив, соответственно называем переменную в множественном числе